tox.ini

```diff
@@ -12,6 +12,8 @@ deps=
   nose<2.0
   unittest2<2.0
   boto<3.0
+  pandas==0.18.0
+  tables
```

Comment:
Any reason why it's so slack?

Perhaps use `<5.0` if the current latest version is `4.x` (for example)